### ISSUE TITLE
Link to GitHub page

### ISSUE THREAD

**Issue by fulldecent** [ethers-io/ethers.io#17]
It should not be so hard to find the project GitHub page.

**Comment by ricmoo** [comment]
Agreed. I will be spending more time on ethers.io, ethers-app and the Web3 bridge, updating it to ethers.js v4, over the next few weeks. I’ll be sure to include a link to the GitHub on the welcome app.

Thanks! :)
